### ISSUE TITLE
Interested in PRs with new enhanced scenarios?

### ISSUE THREAD

**Issue by brookscl** [vasl-developers/scenarios#1]
I've built (and continue to build) a variety of enhanced scenarios for VASL tournaments this year. Are you interested in getting PRs with new submissions?

**Comment by derimmer** [comment]
These would show up on VASL.info correct? Under Scenario setups?

It is not so much a question of our interest as whether or not you would like them to show up there. If you do, then we should be able to make that happen. 

If you know how, go ahead. If you would like help to get started, let us know.

**Comment by brookscl** [comment]
You know... I'm not sure. When I look at the [scenario list](http://vasl.info/scenarios.htm) I see the scenarios in this repo, but I see a lot more. Don't know how they are published.

I also see "Patrick Ireland is currently working on..." there and I believe he passed away about 3 years ago.

**Comment by derimmer** [comment]
I can  help you with the publishing. If you want to learn to do it so that you can do so on an on-going basis, then I am happy to show you how. If it is more of a one-time thing to put up a bunch of setup files, then if you send me the files I would be happy to publish them. 

I don't know how active this part of the repo is. I don't use setup files much myself so i rarely go to the page.

**Comment by brookscl** [comment]
I’m wondering if I should just be doing PRs for here instead: https://github.com/vasl-developers/vasl-website/tree/gh-pages/setups

**Comment by derimmer** [comment]
Yes, that’s exactly how you would do it. That will make them available at vasl.info. Do a test one.
